packages/veritone-react-common/src/components/Parts/Buttons/PillButton/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import defaultStyles from './styles.scss';
+
+export default class PillButton extends Component {
+	static propTypes = {
```

Comment:
Indent should be 2 spaces?  Is prettier setup right?

Reply:
appears not